src/utils/signing.ts

```diff
@@ -71,11 +71,10 @@ export async function signL1Action(
         primaryType: 'Agent',
         message: phantomAgent,
     };
-    return signInner(wallet, data);
+    return signInner(data);
```

Comment:
Here, just return the data so that FE can use it to send for signature to connected wallet.